### ISSUE TITLE
Enabling then disabling music controls breaks watch LED

### ISSUE THREAD

**Issue by bitplane** [MetaWatchOpenProjects/MWM-for-Android-Gen1#2]
1. Open MWM
2. Choose **Menu -> Settings**
3. Check **Music Controls (Beta)**
4. Press back button to sync changes to watch
5. Wait for a couple of seconds to sync changes
6. Open **Menu -> Settings** again
7. Uncheck **Music Controls (Beta)**
8. Press back button to save changes

Expected result:

Bottom left button turns LED on. 
LED can be turned on from watch menu.

Actual result:

Bottom left button doesn't do anything.
LED option in watch menu no longer works until the watch is reset from the menu.

**Comment by garthy** [comment]
It think this is actually an issue in the firmware.

 It only sets up the LED key on startup. 

Once it's reassigned then unassigned it never gets set back to the LED option.

The fix in the firmware is to call DontChangeButtonConfiguration in ConfigureIdleUserInterfaceButtons when returning to normal idle pages.

I'll fix it in my firmware too.

**Comment by MetaWatchOpenProjects** [comment]
Yes -- this is a bug in the APK. The protocol message for assigning the function of the buttons changed and this one fell through the cracks.

I was thinking we would redesign this in the MWM completely, but we haven't gotten around to it. My current idea is to have the music control option take over only one idle mode button (like A) and use the watch Application Mode when started. This way, there isn't any overlap with Idle functions that are useful (like the LED, watch status and menu). This would also give everyone a good application mode usage example (and help us find new watch App mode bugs).
